packages/reusable-blocks/package.json

```diff
@@ -35,6 +35,7 @@
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/icons": "file:../icons",
 		"@wordpress/notices": "file:../notices",
+		"@wordpress/url": "^2.19.0",
```

Comment:
This dependency should be declared as a file link the same as the other `@wordpress/` dependencies above.